doc/manual/building_developing.rst

```diff
@@ -203,7 +204,11 @@ or you can use the more direct version: ::
 
     to see the list of suitable build targets directly.
 
-Any of these commands should produce a directory ``result`` which contains a file ``boot.bin``. As described in :ref:`writing-flash`, if your core device is currently accessible over the network, it can be flashed with :mod:`~artiq.frontend.artiq_coremgmt`. If it is not connected to the network:
+Any of these commands should produce a directory ``result`` which contains a file ``boot.bin``. If your core device is accessible by network, flash with: ::
+
+    $ artiq_coremgmt flash --srcbuild result
```

Comment:
You shouldn't need `srcbuild` for flashing binaries directly contained in the `result`/`results` directory.
`srcbuild` is for directories that has a build tree (e.g. building RISC-V binaries through `python` directly)